index.js

```diff
@@ -1,7 +1,23 @@
 var NgFaker = require('./lib');
 
+const {
+  en,
+  yo,
+  ig,
+  ha
+} = require('./lib/locales');
+
 // create new instance of faker class
-var ngfaker = new NgFaker({ locales: require('./lib/locales') });
+var ngfaker = new NgFaker({
+  locales: {
+    en
+  }
+});
+
+ngfaker.addLocale('yo', yo);
+ngfaker.addLocale('ig', ig);
+ngfaker.addLocale('ha', ha);
```

Comment:
I'm guessing after this point where `ngfaker` is exported, it'll automatically have all the locales and I'm thinking what if the user want only one locale for the dummy data?

Reply:
Maybe it should only add the english locale by default, and should `export` the `yo`, `ig`, `ha` objects, so the user can have the option to add them.

Reply:
I'll find you tomorrow so we can talk this through. Thanks boss!